e2e-tests/tests/infra/appSections/ComponentTreeSection.ts

```diff
@@ -0,0 +1,35 @@
+import { Locator, Page, expect } from "@playwright/test";
+
+/**
+ * A Playwright wrapper around the Component Tree functionality.
+ * TODO: Add method to re-arrange Elements.
+ */
+export default class ComponentTreeSection {
+  private componentTree: Locator;
+  private removeElementButton: Locator;
+
+  constructor(page: Page) {
+    this.componentTree = page.locator(':text("Layers") + ul');
+    this.removeElementButton = page.getByRole("button", {
+      name: "Remove Element",
+    });
+  }
+
+  getElement(name: string, index = 0): Locator {
+    return this.componentTree.getByText(name).nth(index);
+  }
+
+  async setActiveElement(name: string, index = 0) {
```

Comment:
Maybe I missed it, but do we actually pass in an `index` anywhere?

Reply:
I don't think so. I kept it when I cut over the method from `StudioPlaywrightPage`. I could see us potentially using this in the future and it's not adding much code/complexity.